code/scripting/api/objs/object.cpp

```diff
@@ -105,6 +106,9 @@ ADE_FUNC(__tostring, l_Object, NULL, "Returns name of object (if any)", "string"
 		case OBJ_BEAM:
 			sprintf(buf, "%s beam", Weapon_info[Beams[objh->objp()->instance].weapon_info_index].name);
 			break;
+		case OBJ_PROP:
+			sprintf(buf, "%s prop", "TEMP");
```

Comment:
Shouldn't this print the prop's name?

Reply:
Yup. This was almost certainly left over from early work when I was just finding all the relevant switch statements.